src/rubrix/server/elasticseach/client_wrapper.py

```diff
@@ -102,7 +102,8 @@ def list_documents(
         A sequence of documents resulting from applying the query on the index
 
         """
-        return es_scan(self.__client__, query=query or {}, index=index)
+        # Preserve order is used to ensure that the samples are sorted by identifier
+        return es_scan(self.__client__, query=query or {}, index=index, preserve_order=True)
```

Comment:
From elasticsearch client documentation:

> preserve_order – don’t set the search_type to scan - this will cause the scroll to paginate with preserving the order. Note that this can be an extremely expensive operation and can easily lead to unpredictable results, use with caution.

I'm a bit unquiet about the performance impact caused by activating the `preserve_order` flag. If I understand well, this must be set up to fetch the proper last id for the next batch in `rb.load`, right? But, when no limit is applied, we will return the whole dataset, so no need for order preservation, right?

So, for those cases where a limit is provided, I suggest using the search method instead, sorted by the `id` field. Anyway, the better solution is to implement an alternative helper to `es_scan` using automatically the `search_after` attribute. Let me think about a solution that could fit in this PR